test/utils/e2e_flags.go

```diff
@@ -20,4 +20,5 @@ var (
 	TektonPipelineDeploymentLabel  = "app.kubernetes.io/part-of=tekton-pipelines"
 	TektonTriggerDeploymentLabel   = "app.kubernetes.io/part-of=tekton-triggers"
 	TektonDashboardDeploymentLabel = "app.kubernetes.io/part-of=tekton-dashboard"
+	TektonChainsDeploymentLabel    = "app.kubernetes.io/name=tekton-pipelines"
```

Comment:
it should be `"app.kubernetes.io/name=tekton-chains"` :thinking:

Reply:
https://github.com/tektoncd/chains/blob/main/config/100-deployment.yaml#L64

Reply:
we need to bring that up upstream 😝

Reply:
I will create an issue to fix this later